src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/ManagedTypeInfo.cs

```diff
@@ -88,7 +88,7 @@ public sealed record SpecialTypeInfo(string FullTypeName, string DiagnosticForma
         public static readonly SpecialTypeInfo Void = new("void", "void", SpecialType.System_Void);
         public static readonly SpecialTypeInfo String = new("string", "string", SpecialType.System_String);
         public static readonly SpecialTypeInfo Boolean = new("bool", "bool", SpecialType.System_Boolean);
-        public static readonly SpecialTypeInfo IntPtr = new("System.IntPtr", "System.IntPtr", SpecialType.System_IntPtr);
+        public static readonly SpecialTypeInfo IntPtr = new("global::System.IntPtr", "System.IntPtr", SpecialType.System_IntPtr);
```

Comment:
Since we use built-in aliases everywhere else, do we want to instead use `nint`?

Reply:
I need to check the behavior for the nint alias on netstandard2.0. If it behaves the same as on net9.0, I'll update it.